src/backend/aspen/covidhub_import/import_projects.py

```diff
@@ -267,13 +268,19 @@ def import_project(
                 continue
 
             sample = external_accessions_to_samples.get(external_accession, None)
+            uploaded_by_user = (
+                session.query(User).filter(User.name == "ASPEN ADMIN").one()
+            )
             if sample is None:
                 sample = Sample(
-                    submitting_group=group, private_identifier=external_accession
+                    submitting_group=group,
+                    private_identifier=external_accession,
+                    uploaded_by=uploaded_by_user,
```

Comment:
I think you only need the line below since it always runs!

Reply:
I'm confused by what you mean